HtmlEditor: Add identical tag merging notes (#7912)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxHtmlEditor/1 Configuration/value.md

[note]

> - When HTML Editor loads its **value**, the component removes redundant tags:

Reviewer: ```suggestion
- When the HTML Editor loads its **value**, the component removes redundant tags:
```

        <!-- from -->
        <p><span>Hello</span><em><span>World</span></em></p>

        <!-- to -->
        <p>Hello<em>World</em></p>

- The component merges consecutive inline tags that match and contain identical attributes:

        <!-- from -->
        <a href="/">Hello</a><a href="/">World</a>
        
        <!-- to -->
        <a href="/">HelloWorld</a>

>     To avoid this behavior, separate identical inline tags with other tags.

Reviewer: ```suggestion
    To avoid this behavior, separate identical inline tags from other tags.
```

Author: What I meant with "with other tags" here was place some other tag between identical tags. For instance, instead of having two link tags one after the other, place something else in between:

instead of this: `<a>link</a><a>link</a>`
do this: `<a>link</a><span></span><a>link</a>`

If you feel "separate identical inline tags with other tags" is not clear enough, perhaps we can directly advise placing other tags in between identical tags:

```suggestion
    To avoid this behavior, specify other elements between identical inline tags.
```

Reviewer: yes, this is what it should be. your original phrasing didn't relay this. thanks.
